### ISSUE TITLE
[Feature Request]: Notches in BoxPlots

### ISSUE THREAD

**Issue by NikitaKhromov-Borisov** [jasp-stats/jasp-issues#1529]
### Description

In raincloud plots notchces would be desirable, they show 95% Cis for medians

**Comment by Trebicky** [comment]
I would add to this one. Rain clouds are phenomenal. But boxplot have their issues (as far as I understand it https://www.autodesk.com/research/publications/same-stats-different-graphs) and frequently one does report means and CIs for their variables instead of medians etc...

Would it be possible to choose whether I want to see box plots or means plus their CIs (or both), so that my visualisation aligns with data I report?

**Comment by NikitaKhromov-Borisov** [comment]
The best BoxPlots with experimental dots, notches (their width is a CI for
the median) and Cis for means are here
http://shiny.chemgrid.org/boxplotr/

Best regards,
Nikita

чт, 9 дек. 2021 г. в 10:45, Trebicky ***@***.***>:

**Comment by Trebicky** [comment]
Thank you, Nikita. I have used that exact page for some papers, it is nice. But I found it nuisance to go for external tools when it could be a checkbox to tick in JASP GUI. And in my view, it would make sense to have the option to choose whether I see my data better described with box plot or with mean and its CIs (especially, when the plot does not come with a footnote detailing what is visualised, e.g. what is cower by the boxplot.
